No longer available in Linux 2.6.10

- Daniel Drake <dsd@gentoo.org>

diff -urNp svgalib-1.9.19/kernel/svgalib_helper/main.c svgalib-dsd/kernel/svgalib_helper/main.c
--- svgalib-1.9.19/kernel/svgalib_helper/main.c	2004-05-07 10:10:22.000000000 +0100
+++ svgalib-dsd/kernel/svgalib_helper/main.c	2005-01-09 18:25:13.094026296 +0000
@@ -577,7 +577,6 @@ int init_module(void)
 		devfs_mk_cdev(MKDEV(SVGALIB_HELPER_MAJOR, i),
 				S_IFCHR |  S_IRUGO | S_IRWXU, "svga_helper/%d", i);
 	}
-	devfs_mk_symlink("svga", "svga_helper/0");
 # endif
 #endif /* devfsd support */
 
